econml/tests/test_orf.py

```diff
@@ -64,23 +63,24 @@ def test_continuous_treatments(self):
                           TestOrthoForest.X, TestOrthoForest.W)
         # Check that outputs have the correct shape
         out_te = est.const_marginal_effect(TestOrthoForest.x_test)
-        self.assertSequenceEqual((TestOrthoForest.x_test.shape[0], 1), out_te.shape)
+        self.assertEqual(TestOrthoForest.x_test.shape[0], out_te.shape[0])
```

Comment:
Note that this now tests only a single entry rather than the entire shape.

Reply:
Not really, this tests that the lengths are equal since these are now flat arrays rather than column vectors.